extensions/autolink.c

```diff
@@ -9,6 +9,14 @@
 #include <strings.h>
 #endif
 
+// for ssize_t
+#ifdef _MSC_VER
+#include <BaseTsd.h>
+typedef SSIZE_T ssize_t;
```

Comment:
Maybe use `ptrdiff_t` instead? I think that should be defined on Windows.

Reply:
https://github.com/github/cmark-gfm/pull/302/commits/01162c5b28d923bf96472d94821aa3b087b347ef